lib/git.js

```diff
@@ -76,12 +73,11 @@ const commitAndPush = async function(path, branch, gitRoot, logger, action) {
 		rawArgs.push('-m')
 		rawArgs.push('Compile assets')
 	}
-	await git.raw(rawArgs, function(err, result) {
```

Comment:
The raw commit function doesn't return the commit hash anymore. Let's ignore this, we really don't care